.github/workflows/conformance-externalworkloads-v1.10.yml

```
@@ -137,6 +137,16 @@ jobs:
           fi
 
           CILIUM_INSTALL_DEFAULTS="--cluster-name=${{ env.clusterName }} \
+            --chart-directory=install/kubernetes/cilium \
+            --helm-opts=image.repository=quay.io/cilium/cilium-ci \
+            --helm-opts=image.tag=${SHA} \
+            --helm-opts=operator.image.repository=quay.io/cilium/operator \
```

As above, for GKE CI I think this should be:
```suggestion
            --helm-opts=operator.image.repository=quay.io/cilium/operator-generic \
```

---

See https://github.com/cilium/cilium/pull/18898#discussion_r816948873.